change/@ni-nimble-components-ceebd404-0fb3-43e5-8f98-04f4eed64531.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
FYI @mollykreis @jattasNI I mainly did a source code review. I didn't do a deep analysis of if the new interaction matrix stories to see if the subset of states they cover give us confidence in thee coverage or that the stories align with design specs and should be the approved baselines.

Reply:
I did a pass at looking at the coverage of states. It looked mostly good to me. The only thing I wasn't sure about was the usefulness of the "active" (but not "hovered") state for some components. Is there a user interaction that can hit that state? For example, tabbing to a button and pressing Space/Enter doesn't cause it to render in the active state. Maybe clicking on a component and then dragging away so it's no longer hovered? Ultimately I don't have a problem keeping it out of completeness.

Regarding whether the baselines should be approved, I think Mert mentioned he showed this to Brandon earlier but I might be confusing this PR with another one. If not then this would be a good opportunity to do that again (I assume we did it the last time we updated these styles significantly), but I'd be fine with the PR completing before that happens since this isn't changing anything, just snapshotting our current behavior.

Reply:
> Maybe clicking on a component and then dragging away so it's no longer hovered?

I think that might be the only way, but I'm inclined to leave it in.

The one off-spec thing I noticed was some of the anchor styling, though the spec has us using a different color for visited links, and I wonder if we actually want that. There are still a couple other differences that I will address in another PR, and I'll ask Brandon about the visited links.